scripts/bug_retriever.py

```diff
@@ -60,7 +60,15 @@ def retrieve_bugs(self):
 
         # Try to re-download inconsistent bugs, up to three times.
         for i in range(3):
-            bug_ids = set(bug_snapshot.get_inconsistencies())
+            # We look for inconsistencies in all bugs first, then, on following passes,
+            # we only look for inconsistencies in bugs that were found to be inconsistent in the first pass
+            if i == 0:
+                inconsistent_bugs_set = bugzilla.get_bugs()
+            else:
+                inconsistent_bugs_set = inconsistent_bugs
+            inconsistent_bugs = bug_snapshot.get_inconsistencies(inconsistent_bugs_set)
+            bug_ids = set(inconsistent_bugs)
```

Comment:
This one is the variable to rename to `inconsistent_bugs_set` (bug_ids -> inconsistent_bugs_set)

Reply:
Or even better, `inconsistent_bug_ids`